packages/grafana-o11y-ds-frontend/src/TemporaryAlert.tsx

```diff
@@ -0,0 +1,44 @@
+import { css } from '@emotion/css';
+import React, { useEffect } from 'react';
+
+import { Alert, AlertVariant, useTheme2 } from '@grafana/ui';
+
+export const TemporaryAlert = (props: {
+  setVisible: (visible: boolean) => void;
+  severity: AlertVariant;
+  text: string;
+  visible: boolean;
+}) => {
+  const theme = useTheme2();
+
+  useEffect(() => {
+    if (props.visible) {
+      const timer = setTimeout(() => {
+        props.setVisible(false);
+      }, 3000);
```

Comment:
I would recommend a longer display time. I'm sorry I [can't find a better resource](https://sheribyrnehaber.medium.com/designing-toast-messages-for-accessibility-fb610ac364be), but maybe at least 5 seconds. Some people can read slow due to a number of reasons.

Reply:
I like how original component has timeout based on severity. You don't probably need a long timeout for success message, but then for warning/error you do. 
<img width="732" alt="image" src="https://github.com/grafana/grafana/assets/30407135/3c920852-973a-4947-b62e-774c67417cb6">

Reply:
That's fantastic!